client/src/views/profile/AuthenticationPage.vue

```diff
@@ -0,0 +1,145 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="authentication-container">
+    <template v-if="currentDevice">
+      <div class="authentication-content">
+        <authentication-card
+          :image-src="keypadGradient"
+          :image-alt="'keypad icon'"
+          :method-name="'Authentication.method.system'"
```

Comment:
You can just do `image-alt="keypard icon"`. `:` in front of a prop is used to interpret what's between the double-quotes. In this case it's just strings, to `propName="value"` is enough.
Same with most of the other props in this file.